plugins/modules/system/homectl.py

```diff
@@ -0,0 +1,479 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, James Livulpi
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: homectl
+author:
+    - "James Livulpi (@jameslivulpi)"
+short_description: Manage user accounts with systemd-homed
+version_added: 4.4.0
+description:
+    - Manages a user's home directory managed by systemd-homed
+options:
+    name:
+        description:
+            - The user name to create, remove, or modify.
+        required: true
+        aliases: [ 'user', 'username' ]
+        type: str
+    password:
+        description:
+            - Set the user's password to this.
+            - Homed requires this value to be in cleartext on user creation and updating a user.
+            - The module takes the password and generates a password hash in SHA-512 with 10000 rounds of salt generation using crypt.
+            - See U(https://systemd.io/USER_RECORD/).
+            - This is required for I(state=present) and I(state=modify)
+        type: str
+    state:
+        description:
+            - The operation to take on the user such as remove, add user, and modify user.
+        choices: [ 'absent', 'present', 'modify' ]
```

Comment:
Modify is **not** a state. A state is "present" or "absent".

Modification should always be part of "present".

Reply:
I see, I will take care of this.